model/event/event_finder.go

```diff
@@ -83,19 +83,15 @@ func CountUnprocessedEvents() (int, error) {
 // === Queries ===
 
 // Host Events
-func HostEventsForId(id string) db.Q {
+func HostEventsForId(ids []string) db.Q {
 	filter := ResourceTypeKeyIs(ResourceTypeHost)
-	filter[ResourceIdKey] = id
+	filter[ResourceIdKey] = bson.M{"$in": ids}
 
 	return db.Query(filter)
 }
 
-func MostRecentHostEvents(id string, n int) db.Q {
-	return HostEventsForId(id).Sort([]string{"-" + TimestampKey}).Limit(n)
-}
-
-func HostEventsInOrder(id string) db.Q {
-	return HostEventsForId(id).Sort([]string{TimestampKey})
+func MostRecentHostEvents(ids []string, n int) db.Q {
```

Comment:
Not sure how feasible it is, but would you be able to create a function that takes in a host (instead of a host ID) and looks for events for that host ID or tag? That way the caller doesn't need to find out the tag themselves

Reply:
This is feasible 🎉

Reply:
Just kidding -- not feasible because events can't import hosts.